jsonschema-core.xml

```diff
@@ -1121,6 +1143,31 @@
                         validation of core keywords.
                     </postamble>
                 </figure>
+                <t>
+                    "localKeyword" and "startDate" are now valid for any schema that
+                    declares this meta-schema for their "$schema".  Note that defining
+                    it in this way does not imply the new keywords are valid within an
+                    instance to be validated by such a schema.
+                </t>
+                <t>
+                    Although the meta-schema can validate the syntax of "startDate"
+                    within a schema, it is the vocabulary that defines the logic behind
+                    the semantic meaning of "startDate".  Without an understanding of
+                    the semantics (that the instance value should represent a date after that
+                    given in the schema), an implementation can only validate the syntactic
+                    usage of "startDate" (that it is a date-formatted string).  This is why
+                    a schema MUST refuse to validate a schema which uses a meta-schema
```

Comment:
If this is purely informative and not normative (as in it's already covered earlier), then lowercase that `MUST`, imho.

Reply:
✔️